### ISSUE TITLE
Forums: connected accounts doesn't show (linked) discord account

### ISSUE THREAD

**Issue by itVictor** [redsunservers/LoadoutBugTracker#206]
In settings, there's an option for you to link your account with steam and discord. The discord linking is fine, but it doesn't show it.

<img width="319" alt="screen shot 2018-03-31 at 5 10 57 pm" src="https://user-images.githubusercontent.com/20176719/38168001-82b11abe-3506-11e8-966c-b963c2780821.png">

If I click on "associate with discord" I get an error: _This account is already associated with another member. This association can only be changed from within that account._
However this isn't true, as I know I've already linked my discord account to my forum account, and I know that it does work since I can sign into my forum account via discord without any issues. So my discord account IS linked to my forum account, but it doesn't show it in connected accounts (like it does with steam in the screenshot above).

In other words, there's no indication that your discord account is linked to your forum account unless you try signing in with it, and I'm guessing from the screenshot that there's no way to disassociate my discord account because there's no option for it.

**Comment by EgocentricPoliceDick** [comment]
This is probably an issue with the discord linking plugin. There's a few updates for it that have released, so I'll install them and see what changes.

Please report if there is any changes after this.

**Comment by itVictor** [comment]
Sorry for the late reply, but I'm not seeing any changes on my end.

**Comment by EgocentricPoliceDick** [comment]
Yes, I'm currently working on other issues right now and haven't gotten around to plugin enhancements for the forums just yet - see here: https://steamcommunity.com/groups/redsunoverparadise#announcements/detail/1644251867797607966

**Comment by EgocentricPoliceDick** [comment]
Alright, I've put an upgrade through for the associated plugin. Report back if anything has changed.

**Comment by itVictor** [comment]
Looks good to me, didn't have any issues with dissociating or signing in. I'll go ahead and close this issue.

<img width="321" alt="screen shot 2018-04-02 at 6 29 58 pm" src="https://user-images.githubusercontent.com/20176719/38221214-fc7f89f4-36a3-11e8-8e29-a69ca086ea74.png">
